lib/Perl/Critic/Command.pm

```diff
@@ -518,7 +518,7 @@ sub _get_option_specification {
 
 sub _colorize_by_severity {
     my @violations = @_;
-    return @violations if _this_is_windows();
+    return @violations if _this_is_windows() && !eval 'require Win32::Console::ANSI; 1';
```

Comment:
@dolmen When you say "more efficient", do you mean it executes faster?  How much faster?